phpcs.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for Co-Authors Plus">
+
+	<rule ref="WordPress-VIP" />
+	<rule ref="WordPress-Docs" />
+	<rule ref="WordPress-Extra" />
+
+	<arg value="s"/>
```

Comment:
Consider adding flags to show progress, use shorter basepath, adding colors etc. The exact args available depends on what PHPCS version is installed, and that depends on what minimum PHP version should be supported.